Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h

```diff
@@ -3860,7 +3860,1103 @@ e.g. remove mangling for H5Fget_info but leave it for H5Fget_info1 and H5Fget_in
 #define HDstrtoll itk_HDstrtoll
 #define Nflock itk_Nflock
 #define Pflock itk_Pflock
-#define epoch_marker_class itk_epoch_marker_class
+#define _ZN2H510AbstractDsC2Ev itk__ZN2H510AbstractDsC2Ev
```

Comment:
I think that symbols starting with `_` are belonging to the standard library, and should not be on the mangle list.

Reply:
Good to know. I will remove these, then, and add that note to the update instructions.